src/ResourceManager/Automation/Commands.Automation/help/Import-AzureRMAutomationRunbook.md

```diff
@@ -21,11 +21,10 @@ Import-AzureRmAutomationRunbook [-Path] <String> [-Description <String>] [-Name
 ```
 
 ## DESCRIPTION
-The **Import-AzureRmAutomationRunbook** cmdlet imports an Azure Automation runbook. Specify the
-path to a wps_2 script (.ps1 ) file to import for wps_2 and wps_2 Workflow runbooks, or to a
-graphical runbook (.graphrunbook) file for graphical runbooks. (.py) files are required when importing python2 scripts The name of the file becomes the
-name of the runbook. For wps_2 Workflow runbooks, the script must contain a single wps_2 Workflow
-definition that matches the name of the file.
+The **Import-AzureRmAutomationRunbook** cmdlet imports an Azure Automation runbook. Specify the 
+path to a wps_2 script (.ps1) file to import for wps_2 and wps_2 Workflow runbooks, 
+or to a graphical runbook (.graphrunbook) file for graphical runbooks, or to a python 2 script (.py) file for python 2 runbooks. 
```

Comment:
Can you remove the 'or' before the 'to a graphical runbook' please? Sorry I might have added it in my suggestion but 2 'or' doesnt make sens.
Singing off.

Thanks!